WordPress/src/main/java/org/wordpress/android/ui/comments/NotificationCommentDetailFragment.kt

```diff
@@ -26,6 +30,27 @@ class NotificationCommentDetailFragment : CommentDetailFragment() {
         }
     }
 
+    override fun getUserProfileUiState(): BottomSheetUiState.UserProfileUiState {
+        val user = mContentMapper.mapToFormattableContentList(mNote!!.body.toString())
+            .find { FormattableRangeType.fromString(it.type) == FormattableRangeType.USER }
+
+
+        return BottomSheetUiState.UserProfileUiState(
+            userAvatarUrl = mNote!!.iconURL,
+            blavatarUrl = "",
+            userName = user?.text ?: getString(R.string.anonymous),
+            userLogin = mComment?.authorEmail ?: "",
```

Comment:
Super nit: here and in other places `.orEmpty()` function may look nicer (:
```suggestion
            userLogin = mComment?.authorEmail.orEmpty(),
```

Reply:
Yeah, it looks more elegant, I just updated.